.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: NodeJS with Webpack
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
I don't run on different node version s anymore. The versions pretty much behave the same at this point so it simplified the setup.